README.md

```diff
@@ -65,10 +65,6 @@ This example shows a request document to create a post and a related tag in a si
             {
               "type": "tags",
               "id": "7c237585-983e-4767-a425-5f2277ba7351"
-            },
-            {
-              "type": "tags",
-              "lid": "2"
             }
```

Comment:
I find this hard to understand. Since PATCH requests do a full replacement of one-to-many relationships.

I understand it works because the document needs to be processed top-to-bottom. But that's something else than easy to wrap one's head around it.